src/agent/state.js

```diff
@@ -290,7 +296,16 @@ StateResolver.prototype.isPathInNodeModulesDirectory_ = function(path) {
 StateResolver.prototype.resolveFrame_ = function(frame, underFrameCap) {
   var args = [];
   var locals = [];
-  if (!underFrameCap) {
+  if (!this.config_.allowExpressions) {
+    args.push({
+      name: 'arguments_not_available',
+      varTableIndex: EXPRESSIONS_DISALLOWED_INDEX
+    });
+    locals.push({
+      name: 'locals_not_available',
```

Comment:
After digging back through some of the frame mirror resolution logic, I think locals/args are fine. I've added a comment justifying this.